kie-osgi/kie-karaf-itests-kjar/pom.xml

```diff
@@ -14,4 +14,12 @@
     <maven.deploy.skip>true</maven.deploy.skip>
   </properties>
 
+  <dependencies>
+    <dependency>
+      <groupId>org.kie</groupId>
+      <artifactId>kie-karaf-itests-domain-model</artifactId>
+      <version>7.0.0-SNAPSHOT</version>
```

Comment:
At the moment we cannot merge this PR because the script that updates the versions numbers when a new release has to be dropped doesn't take count of this. We need to fix that script first.